# Suora johdin ja käämi magneettikentässä

In [1]:
using Printf
using Dates
@printf "%s\n" Dates.now();
versioninfo(verbose=false);

2021-10-12T18:24:15.774
Julia Version 1.5.4
Commit 69fcb5745b (2021-03-11 19:13 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, skylake)
Environment:
  JULIA_DEPOT_PATH = C:\Users\snikula\.julia;C:\Users\snikula\AppData\Local\JuliaPro-1.5.4-1\Julia-1.5.4\local\share\julia;C:\Users\snikula\AppData\Local\JuliaPro-1.5.4-1\Julia-1.5.4\share\julia
  JULIA_LOAD_PATH = C:\Users\snikula\.julia\environments\JuliaPro_v1.5.4-1;@;@v#.#;@stdlib
  JULIA_PKG_SERVER = pkg.juliahub.com


8-4

Kirjan vastaus: a) 200 μT b) 20 μT c) 2 μT d) 2.6 A

8-5

Kirjan vastaus: a) 0.13 mN ylös b) 15 mT

8-6

Kirjan vastaus: 1.5 T

8-7

Kirjan vastaus: 6.7 mN vasemmalle

8-8

Kirjan vastaus: b) 33 mA oikealle

8-9

Kirjan vastaus: b) 37 mA

8-10

Kirjan vastaus: b) 9.2e-6 N vetovoima

8-11

Kirjan vastaus: 3.9 Nm